src-electron/generator/matter/chip-tool/templates/helper.js

```diff
@@ -163,13 +163,27 @@ async function assertSameTestType(current, expected) {
   throw error;
 }
 
+const dep = templateUtil.deprecatedHelper;
+
 //
 // Module exports
 //
-exports.asDelimitedCommand = asDelimitedCommand;
-exports.asTypeMinValue = asTypeMinValue;
-exports.asTypeMaxValue = asTypeMaxValue;
-exports.structs_with_cluster_name = structs_with_cluster_name;
+exports.asDelimitedCommand = dep(
+  asDelimitedCommand,
+  'asDelimitedCommand has been deprecated. Use cleanse_label_as_kebab_case instead'
+);
+exports.asTypeMinValue = dep(
+  asTypeMinValue,
+  'asTypeMinValue has been deprecated. Use as_underlying_zcl_type_size_bits, if_is_underlying_zcl_type_signed and if_is_float_or_double_data_type instead'
+);
+exports.asTypeMaxValue = dep(
+  asTypeMaxValue,
+  'asTypeMaxValue has been deprecated. Use as_underlying_zcl_type_size_bits, if_is_underlying_zcl_type_signed and if_is_float_or_double_data_type instead'
+);
```

Comment:
This forces a lot of complicated logic on consumers, as it turned out in the Matter-side PR....  We need a better solution here.